packages/common/container-definitions/src/runtime.ts

```diff
@@ -126,11 +126,15 @@ export interface IBatchMessage {
 
 /**
  * IContainerContext is fundamentally just the set of things that an IRuntimeFactory (and IRuntime) will consume from the
- * loader layer.  It gets passed into the IRuntimeFactory.instantiateRuntime call.  Only include members on this interface
- * if you intend them to be consumed/called from the runtime layer.
+ * loader layer.
+ * It gets passed into the {@link (IRuntimeFactory:interface).instantiateRuntime} call.
+ *
+ * @privateremarks
+ * Only include members on this interface if you intend them to be consumed/called from the runtime layer.
+ *
+ * TODO: once `@alpha` tag is removed, `unknown` should be removed from submitSignalFn.
+ * See {@link https://dev.azure.com/fluidframework/internal/_workitems/edit/7462}
```

Comment:
I couldn't get this to be reflected in the actual docs, might be just lack of familiarity with _how_ to make it happen. Any pointers welcome.

Reply:
Sent you some details on Teams. A short-term workaround for this limitation is in PR, and I'm trying to get work scheduled for a longer-term solution.